### DISCUSSION TITLE
Slide Show for this week

### DISCUSSION THREAD

**Discussion by RuthAnn1** [Techwise-Team-15/my_pet#100] [General]
Here is the slide show for this week.  I tried a new thing and sent it to everyone's email but I am also adding the slide show here.  Let me know if you don't have access to it.

https://docs.google.com/presentation/d/1fQOe_R_iSUSxYVNVM8HT-P9Gt71PrdBlbffTVsfOIaM/edit?usp=sharing

**Comment by dinaolmelak** [comment]
Thank you 👍 , I have gone ahead and pinned this discussion @RuthAnn1
